builder/amazon/instance/builder_test.go

```diff
@@ -26,6 +26,8 @@ func testConfig() map[string]interface{} {
 		"x509_key_path":    tf.Name(),
 		"x509_upload_path": "/foo",
 	}
+
+	return config, tf.Name()
```

Comment:
looks like tf.Name() is already returned within the config as "x509_key_path" -- do we need to add a separate return value for it?

Reply:
You know what? I like your way better. It's more obvious to read.